composer.json

```diff
@@ -14,12 +14,12 @@
         }
     ],
     "require": {
-        "php": ">=5.3"
+        "php": "^7.4"
```

Comment:
Most tend to do >=version atm...

So `>=7.4` would be fine (though, I would prefer `>=7.2`)

Or `^7.4||^8.0` would be better (or well, `^7.2||^8.0`)